ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74__ocp-4-13.yaml

```diff
@@ -0,0 +1,71 @@
+base_images:
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.57
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.13.0-0
+        upper: 4.14.0-0
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- as: qa-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: openshift-4-qa-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- as: operator-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: openshift-4-operator-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- as: merge-qa-e2e-tests
+  postsubmit: true
+  steps:
+    cluster_profile: aws
+    env:
+      QA_TEST_TARGET: test
```

Comment:
What does `QA_TEST_TARGET: test` provide?

Reply:
This is my understand: in [test_part_1](https://github.com/stackrox/stackrox/blob/master/qa-tests-backend/scripts/run-part-1.sh#L75-L116), we don't have a check to determine if the script is being run in a merge scenario but we want to run `make -C qa-tests-backend test`, and adding `QA_TEST_TARGET: test` will run that command

@gavin-stackrox might be able to give a better or more compressive answer 🙂

Reply:
Thanks, I did not realize that `QA_TEST_TARGET` is for stackrox.
I think it opens a bigger question why don't we have `test` as default in all post-submits.